.config/dotnet-tools.json

```
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "kinsondigital.cicd": {
+      "version": "1.0.0-preview.20",
```

Update this to the latest version.
There have been some releases since you worked on this issue.

```suggestion
      "version": "1.0.0-preview.22",
```